### PR TITLE
WIP - adding more contribution information

### PR DESCRIPTION
**Pull request by joel-hamill**

### CODE REVIEW

**Comment by mhausenblas** [comment]
Thanks @joel-hamill — can you or @sascala fix one more thing pls: it still mentions raising a GitHub issue, while we're trying to close down this path and migrate to [Jira](https://dcosjira.atlassian.net) (with the `dcos-docs` component). So, can you pls update this in step 1 of _Make your update_?

**Comment by joel-hamill** [comment]
@sascala ready for merge

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
+Use these guidelines to contribute to the DC/OS documentation.
 
-1. Create a [new issue](https://github.com/dcos/dcos-docs/issues) in this repo (`dcos-docs`) and label it with the type of documentation that you're writing. For example, [tutorial](https://github.com/dcos/dcos-docs/labels/tutorial).
+## Format guidelines
+
+Markdown in this repository is formatted for rendering by using [Metalsmith](http://www.metalsmith.io/).
+
+- Links must include the full directory path, including version, relative to the root of dcos.io (e.g. `/docs/1.7/administration/sshcluster/`). Note that these links will not work in the GitHub code browser. It is recommended that you test your content [locally](#test-local) before submitting your PR.
+- Final page links are directory names, not filenames (e.g. `https://dcos.io/docs/latest/usage/service-discovery/mesos-dns/`).
+- You must have an `index.md` page for all parent directories (rather than using Github's README.md indexing). For example, the parent directory `/dcos-docs/1.7/administration/` must also contain `/dcos-docs/1.7/index.md`.
```
Comment (pyronicide) [inline CONTRIBUTING.md]: `/dcos-docs/1.7/administration/index.md`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
+- The table of contents of each page is automatically generated based on the top-level headers.
+- Directory tables of contents are automatically generated based on `post_title` and `post_excerpt` headers.
```
Comment (pyronicide) [inline CONTRIBUTING.md]: Isn't it `nav_title`?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -25,16 +40,51 @@ Use these guidelines to contribute to the DC/OS documentation:
...
+    ```
+    npm install
+    ```
+1. Launch local dev server:
+
+    ```
+    npm start
+    ```
+    (opens dev server in browser)
+
```
Comment (pyronicide) [inline CONTRIBUTING.md]: We need the instructions that explain how to see what's current in your `dcos-docs` checkout.

File: CONTRIBUTING.md (hunk 2 above)
Comment (mhausenblas) [inline CONTRIBUTING.md]: Yes @pyronicide it's `nav_title`

Comment (joel-hamill) [inline CONTRIBUTING.md]: @pyronicide @mhausenblas what if there isn't a nav_title?

Comment (mhausenblas) [inline CONTRIBUTING.md]: Then, @joel-hamill, the `post_title` is used for the nav title.

**Comment by karlkfi** [comment]
testing the re-enabled travis build on this PR... seems to be broken

**Comment by karlkfi** [comment]
fixed the travis build

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -76,6 +76,17 @@ If you're unsure about what exactly should go into the tutorial, you can always
     ```
     $ git submodule update --init --recursive
     ```
+1.  Pull in the changed content from your local branch of `dcos/dcos-docs` to the `dcos/dcos-website/dcos-docs` subdirectory. By default, the `dcos-website` project contains the upstream `dcos-docs` submodule of the live site. You can pull in this content by using either of these methods:
```
Comment (joel-hamill) [inline CONTRIBUTING.md]: @emanic take a look at this ^

**Comment by pyronicide** [comment]
@joel-hamill where are we here?

File: CONTRIBUTING.md (hunk 4 above)
Comment (emanic) [inline CONTRIBUTING.md]: OK let me test it out and see if this works.

**Comment by joel-hamill** [comment]
@pyronicide merged

**Pull request merged**